wallet/src/params.rs

```diff
@@ -76,4 +77,17 @@ impl NodeParams {
             }
         }
     }
+
+    /// Get the address of an existing JsonRpcClient actor.
+    pub fn get_client(&self) -> NodeClient {
+        let lock = self.client.clone();
+        log::trace!(
+            "Getting JsonRpcClient actor from worker ({} references)",
+            Arc::strong_count(&lock)
+        );
+
+        lock.read()
+            .map(|x| x.clone())
+            .expect("Node client lock should never be poisoned")
```

Comment:
This may be true because locks can only be poisoned on write, and I can't find where `self.client` is written to. But if it's not written to, you should be able to remove the `RwLock`.

Reply:
Yes, at this point it can be removed. It was there in the first place because I implemented a way for the client address to be swapped in case the `JsonRpcClient` actor died, but I removed that later because it didn't work well enough.